_component_design/table-expansion.md

```diff
@@ -0,0 +1,344 @@
+---
+title: Table Expansion
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Table rows can be expanded to show the user additional data. By hiding some table data that can only be accessed at the request of the user, expanded tables provide for more data rich layouts without increasing the complexity for the user.
+preview-image: preview-images/tables-nested.svg
+status: stable
+resource: true
+last-modified: 2018-12-03
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use expanded tables when you want to include additional levels of data within a table.
```

Comment:
smaller data set cause don't need to load it all. Use when you want data accessible but you don't need a available all the time. performance considerations.

Reply:
kvp: is tabular data.

Reply:
Done.